src/splitInteger.test.js

```diff
@@ -4,18 +4,23 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+    expect(splitInteger(6,2))
+    .toEqual([3,3])
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+    expect(splitInteger(8,1))
+    .toEqual([8])
 });
 
 test('should sort parts ascending if they are not equal', () => {
-
+  expect(splitInteger(17,4))
+  .toEqual([4,4,4,5])
 });
 
 test('should add zeros if value < numberOfParts', () => {
+  expect(splitInteger(2,3))
+  .toEqual([0,0,1])
```

Comment:
It looks like there was a misunderstanding in the expected output in the test you provided. The expected output should actually be [0, 0, 2] instead of [0, 0, 1], because when splitting 2 into 3 parts, two parts will be 0 and the last part will be 2.